### ISSUE TITLE
Wrong link target for "plugin hook"

### ISSUE THREAD

**Issue by E-VANCE** [pluginkollektiv/antispam-bee#45]
Hi there,

very much appreciate you guys taking over and maintaining the plugin, has done wonders so far with regards to the spam rate on my site(s).

On the settings screen there is a link on "Predefined and custom patterns by plugin hook" which points towards https://gist.github.com/4242142 but throws a 404.

I don't know what the target might be so a pull / fix is not possible.

Thanks for taking a look & keep it up.
Regards

**Comment by Zodiac1978** [comment]
Unfortunately the original plugin author has deleted the gist before we can save it :(

There is already a issue for all broken links:
https://github.com/pluginkollektiv/antispam-bee/issues/38

We try to get this info back online. Please be patient. Thank you!

All the best,
Torsten

**Comment by swissspidy** [comment]
Closing this in favour of #38.
